sdk/internal/recording/recordings/TestUriSanitizer.json

```diff
@@ -0,0 +1,60 @@
+        "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
+        "Feedback.Bootstrap.InitializeFeedback({page:true},\u0022sb_feedback\u0022,1,0,0);;\n",
+        "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
+        "_G!==undefined\u0026\u0026_G.EF!==undefined\u0026\u0026_G.EF.bmasynctrigger===1\u0026\u0026window.requestAnimationFrame!==undefined\u0026\u0026document.visibilityState!==undefined\u0026\u0026document.visibilityState===\u0022visible\u0022?requestAnimationFrame(function(){setTimeout(function(){BM.trigger()},0)}):BM.trigger();\n",
+        "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
+        "var ShareDialogConfig ={\u0022shareDialogUrl\u0022:\u0022/shared/sd/?IID=SERP.5051\u0022};;\n",
+        "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
+        "var wlc=function(n,t,i){var u,f,r;n\u0026\u0026Identity\u0026\u0026(u=Identity.popupLoginUrls)\u0026\u0026(f=u.WindowsLiveId)\u0026\u0026Identity.wlProfile\u0026\u0026(r=_d.createElement(\u0022iframe\u0022),r.style.display=\u0022none\u0022,r.src=f\u002B\u0022\u0026checkda=1\u0022,r.setAttribute(\u0022data-priority\u0022,\u00222\u0022),_d.body.appendChild(r),i\u0026\u0026t\u0026\u0026t(\u0022SRCHHPGUSR\u0022,\u0022WTS\u0022,i,1,\u0022/\u0022))};\n",
+        "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
+        "(function() { var conditionalSignInParams ={\u0022notifEnabled\u0022:false,\u0022notifFetchAsync\u0022:false}; BingAtWork.ConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn(conditionalSignInParams); })();;(function() { var config ={\u0022url\u0022:\u0022https%3a%2f%2flogin.microsoftonline.com%2fcommon%2foauth2%2fauthorize%3fclient_id%3d9ea1ad79-fdb6-4f9a-8bc3-2b70f96e34c7%26response_type%3did_token%2bcode%26nonce%3d2ffee435-6f0f-4238-8eec-12b296d9638c%26redirect_uri%3dhttps%253a%252f%252fwww.bing.com%252forgid%252fidtoken%252fconditional%26scope%3dopenid%26response_mode%3dform_post%26msafed%3d0%26prompt%3dnone%26state%3d%257b%2522ig%2522%253a%252281D4A8D7A8BC437CA9971ACDE3401706%2522%257d\u0022,\u0022sandbox\u0022:\u0022allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts\u0022,\u0022currentEpoch\u0022:\u00221629906927000\u0022}; sj_evt.fire(\u0027ssoFrameExists\u0027, config); })();;\n",
+        "//]]\u003E\u003C/script\u003E\u003C/div\u003E\u003C/body\u003E\u003C/html\u003E"
+      ]
+    }
+  ],
+  "Variables": {}
+}
```

Comment:
```suggestion
}

```

Reply:
These are generated by the test-proxy so I am inclined to leave them as is. I'll reach out to scott B about what would go in here.